File: app/src/main/java/com/lctapp/lct/Classes/Models/Member/MemberProgramme.java
package com.lctapp.lct.Classes.Models.Member;

import java.util.List;

public class MemberProgramme {
    public String programmeCode;
    public int programmeId;
    public String programmeName;
    public List<Voucher> vouchers;
    public int schemeId;
    public boolean isActive;
    public Object insuranceCode;
    public Object schemeCode;

    public String getProgrammeCode() {
        return programmeCode;
    }

    public int getProgrammeId() {
        return programmeId;
    }

    public String getProgrammeName() {
        return programmeName;
    }

    public List<Voucher> getVouchers() {
        return vouchers;
    }

    public int getSchemeId() {
        return schemeId;
    }

    public boolean isActive() {
        return isActive;
    }

    public Object getInsuranceCode() {
        return insuranceCode;
    }

    public Object getSchemeCode() {
        return schemeCode;
    }
}

File: app/src/main/java/com/lctapp/lct/Classes/Models/MemberBalances/MemberProgramme.kt
package com.lctapp.lct.Classes.Models.MemberBalances

import java.io.Serializable

class MemberProgramme: Serializable {
    var programmeCode: String? = null
    var programmeId = 0
    var programmeName: String? = null
    var vouchers: List<Voucher>? = null
    var schemeId = 0
    var isActive = false
    var insuranceCode: Any? = null
    var schemeCode: Any? = null
}